input/config/ubuntu/22.04/kubeflow.json

```diff
@@ -351,9 +420,14 @@
      },
      {
          "package": "quay.io/aipipeline/pipelineloop-webhook",
-         "tag": "1.9.2",
+         "tag": "nightly",
```

Comment:
Shouldn't this be pinned to a version?

Reply:
refer - https://github.com/kubeflow/manifests/blob/v1.9.1-rc.2/apps/kfp-tekton/upstream/third-party/tekton-custom-task/pipeline-loops/500-webhook.yaml